### ISSUE TITLE
Online dmp

### ISSUE THREAD

**Issue by tony1994513** [studywolf/pydmps#2]
Dear Mr.studywolf,
I am a master student and tried using DMP as a way to generate a new trajectory when the robot is detected hit by someone. The robot I am using is Rethink baxter dual arms robot. I want to do online DMP just as peter pastor did in this vedio https://www.youtube.com/watch?v=Ge0GduY1rtE 
Did you have such experience to make it online? The difficulty I met is when I give robot a whole trajectory, it will finish the trajectory then to the next one, which means the robot will stop for a while, or you can say not smooth.
I really appreciate if you can take time to look at my question. Thankyou!

**Comment by studywolf** [comment]
Hello! 

Hmmm i'm not quite sure I understand. Do you mean you're having trouble reproducing the part where they move the target as the robot is moving?
Sorry about the confusion, can you rephrase for me?

**Comment by tony1994513** [comment]
Dear Mr.studywolf,
Sorry for making you feel confusing. Yes, that's what I mean. Have you even done such experiments?
Thank you so much!

**Comment by studywolf** [comment]
Hello, yup! In fact, this is built in to the DMP system naturally. To have this work, all you need to do is run the DMPs forward a step at a time and change the `goal` during the run. I've added an example now that shows what I mean in code here: https://github.com/studywolf/pydmps/blob/master/examples/moving_target_draw_number_2.py

let me know if this makes sense!

**Comment by tony1994513** [comment]
Yes, That's wonderful. Thank you so much!

**Comment by tony1994513** [comment]
Yes, that's wonderful. Thank you so much!

**Comment by studywolf** [comment]
you're very welcome!
